portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,20 +30,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet { 
 
-  ArrayList<String> cmtarr = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
-    String json = new Gson().toJson(cmtarr);
-    response.getWriter().println(json);
+    //String json = new Gson().toJson(cmtarr);
```

Comment:
doGet is not complete right?

Reply:
Yes, these commits were only for storing comments in the datastore